From 0fb698281c08fe18e3edfd5c7a95f9f1f8a8459e Mon Sep 17 00:00:00 2001
From: Jocelyn Falempe <jocelyn.falempe@intel.com>
Date: Tue, 22 Jul 2014 09:29:33 +0200
Subject: [PATCH 7/7] [PATCH] Dirty workaround for buildbot

xxx-user target is broken, and buildbot can't handle that.
dirty patch to make buildbot happy

Change-Id: Ie464363963df4618cb95c386f38914c2b639324a
---
 core/main.mk |   15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/core/main.mk b/core/main.mk
index cdf3200..4c328e2 100644
--- a/core/main.mk
+++ b/core/main.mk
@@ -1,3 +1,16 @@
+# user target is broken, and buildbot can't handle that.
+# dirty patch to make buildbot happy
+ifeq ($(TARGET_BUILD_VARIANT),user)
+.PHONY: droid dist publish_ci flashfiles
+droid:
+	@echo "Dummy target to workaround Buildbot's lack of flexibility"
+dist:
+	@echo "Dummy target to workaround Buildbot's lack of flexibility"
+publish_ci:
+	@echo "Dummy target to workaround Buildbot's lack of flexibility"
+flashfiles:
+	@echo "Dummy target to workaround Buildbot's lack of flexibility"
+else
 # Only use ANDROID_BUILD_SHELL to wrap around bash.
 # DO NOT use other shells such as zsh.
 ifdef ANDROID_BUILD_SHELL
@@ -1061,3 +1074,5 @@ showcommands:
 .PHONY: nothing
 nothing:
 	@echo Successfully read the makefiles.
+
+endif
-- 
1.7.10.4

